packages/kbn-alerts-as-data-utils/src/schemas/generated/ml_anomaly_detection_schema.ts

```diff
@@ -71,7 +71,7 @@ const MlAnomalyDetectionAlertRequired = rt.type({
 });
 // prettier-ignore
 const MlAnomalyDetectionAlertOptional = rt.partial({
-  'kibana.alert.anomaly_score': schemaNumber,
+  'kibana.alert.anomaly_score': schemaNumberArray,
```

Comment:
For the alerts table in the Anomaly Explorer, what is now being displayed in the `Latest anomaly score' column? Looks like it's the first item in the `kibana.alert.anomaly_score` array. Shouldn't it be the last item?

Reply:
Fixed iin https://github.com/elastic/kibana/pull/174919/commits/653a79443a38407c33c1c6080f513454a95d7f8c

Reply:
The value in the alerts table looks correct, but the alert details flyout looks like it is still showing the first item from the `kibana.alert.anomaly_score` array.

Reply:
Confirmed fixed in https://github.com/elastic/kibana/pull/174919/commits/7997ea605e7d3cd4afd477210ee66ddc3806c18f.